CMSgov/managed-care-review
# MR-2844 Rates Across Subs should use radio buttons

## Summary
Use radio buttons instead of checkbox on rates across subs. Also add design tweaks for summary page work.

#### Related issues
https://qmacbis.atlassian.net/browse/MR-2844
See design tweak comments on https://qmacbis.atlassian.net/browse/MR-2686

#### Screenshots
![Screen Shot 2022-12-05 at 5 15 13 PM](https://user-images.githubusercontent.com/10750442/205766158-1d09ff00-c51b-4589-85c1-400076911036.png)
![Screen Shot 2022-12-05 at 5 12 25 PM](https://user-images.githubusercontent.com/10750442/205766216-0712245a-872e-49ad-a31c-53d6bf97fa9e.png)

#### Test cases covered
- No new tests, had to fix tests to work for the radio buttons
- Our tests are really long in RateDetails, we need to simplify. Felt like this was out scope for this ticket but this is also the test we have periodic timeouts in unit tests because we are doing a lot. 

## QA guidance
- There should be no change in the data stored or behaviors on edit. The radio button is UI only and not stored as a field in db, we just store the programs list in the associated dropdown that appears when you click "Yes".
- Validations state work checking, for yes, no, yes with the dropdown selecton, yes with dropdown.
